en/docs/reference/mediators/cache-mediator.md

```diff
@@ -156,90 +156,102 @@ Following are examples of how you can use the Cache mediator.
+    <proxy xmlns="http://ws.apache.org/ns/synapse" name="cache115" transports="http https" startOnLoad="true">
+       <description />
+       <target>
+          <inSequence>
+             <cache collector="false" timeout="60">
+                <protocol type="HTTP">
+                   <methods>POST</methods>
+                   <headersToExcludeInHash />
+                   <responseCodes>.*</responseCodes>
+                   <enableCacheControl>false</enableCacheControl>
+                   <includeAgeHeader>false</includeAgeHeader>
+                   <hashGenerator>org.wso2.carbon.mediator.cache.digest.HttpRequestHashGenerator</hashGenerator>
+                </protocol>
+             </cache>
+             <call>
+                <endpoint key="SomeServiceEndpoint"/>
+             </call>
+             <property name="RESPONSE" value="true" scope="default" type="STRING" />
+             <log level="full" />
+             <cache collector="true" />
+             <property name="RESPONSE" value="false" scope="default" type="STRING" />
+             <cache collector="false" timeout="60">
+                <protocol type="HTTP">
+                   <methods>POST</methods>
+                   <headersToExcludeInHash />
+                   <responseCodes>.*</responseCodes>
+                   <hashGenerator>org.wso2.carbon.mediator.cache.digest.HttpRequestHashGenerator</hashGenerator>
+                </protocol>
+             </cache>
+             <call>
+                <endpoint key="HelloServiceEndpoint"/>
+             </call>
+             <property name="RESPONSE" value="true" scope="default" type="STRING" />
+             <log level="full" />
+             <cache collector="true" />
+             <respond />
+          </inSequence>
+       </target>
+    </proxy>          
+    ```
+=== "Endpoint 1"
+    ``` xml
+    <endpoint xmlns="http://ws.apache.org/ns/synapse" name="SomeServiceEndpoint">
+        <address uri="http://demo0585968.mockable.io/some"/>
+    </endpoint>
+    ```
+=== "Endpoint 2"
+    ``` xml
+    <endpoint xmlns="http://ws.apache.org/ns/synapse" name="HelloServiceEndpoint">
+        <address uri="http://demo0585968.mockable.io/hello"/>
+    </endpoint>
+    ```
 
 ### Example 2
 
-According to this example configuration, when the first message is sent
-to the endpoint, the cache is not hit. The Cache mediator configured in
-the `         Out        ` sequence caches the response to this message.
-When a similar message is sent to the endpoint for the second time, the
-previous response is directly fetched from the cache and sent to the
-requester. This happens because the `         onCacheHit        `
-sequence is not defined in this configuration.
+In this example configuration, when the first message is sent to the endpoint, 
+the cache is not hit because it is the initial request and the cache is empty. 
+The Cache mediator, configured in the main sequence, caches the response to this message. 
```

Comment:
```suggestion
The Cache mediator, configured in the inSequence, caches the response to this message. 
```

Reply:
Shall I wrap this sequence inside an API? I left the sequence's name as main, Shall I change it to avoid confusions with previously existed main sequence?